src/components/event-card/EventCard.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+    <div
+        class="card vs-event-card"
+        data-test="vs-event-card"
+    >
+        <div class="card-body">
+            <div
+                class="vs-event-card__header"
```

Comment:
Can this layout be built with our grid rather than custom flexbox? https://visitscotland.atlassian.net/wiki/spaces/DE/pages/296353794/Grid